apps/backend/package.json

```diff
@@ -37,14 +37,12 @@
 		"pino": "9.3.2",
 		"pino-pretty": "11.2.2",
 		"shared": "*",
-		"swagger-jsdoc": "6.2.8",
-		"ts-node": "10.9.2"
+		"swagger-jsdoc": "6.2.8"
 	},
 	"devDependencies": {
 		"@types/bcrypt": "5.0.2",
 		"@types/convict": "6.1.6",
 		"@types/swagger-jsdoc": "6.0.4",
-		"ts-node": "10.9.2",
```

Comment:
Add back ts-node to devDependencies to avoid issues with migrations as we had before